data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -142,7 +181,7 @@ _ "This attack uses 6 attacks"#enddef
 
             [set_variable]
                 name = consecutive_attacks
-                value = 0
+                value = 1
```

Comment:
Documentation would help here. I'd expect `1` to mean that the unit has already attacked once, but that doesn't fit a recruit or recall event.

Reply:
The comment was written when this code was `name=consecutive_attacks` and `value=1`, but renaming it to `current_verse` makes a different issue. The edge-case of a newly spawned unit having this variable default to 0 was previously handled by that meaning that the first verse should be used, it now means no verse can be used.